### ISSUE TITLE
All openapi-generator* artifacts missing from Maven Central.

### ISSUE THREAD

**Issue by jimklo** [OpenAPITools/openapi-generator#153]
##### Description

No `org.openapitools` group artifacts in Maven Central.

Hence all versions and plugins for openapi-generator are missing from Maven Central.

##### openapi-generator version

ALL

##### OpenAPI declaration file content or url

not applicable

##### Command line used for generation

not applicable

##### Steps to reproduce

```bash
curl https://repo.maven.apache.org/maven2/org/openapitools
```

##### Related issues/PRs

none that I could find

##### Suggest a fix/enhancement

Someone publishes to artifacts to Maven central...

**Comment by jmini** [comment]
There will be there when we have published our first release. ;-)

This should be the case very soon (1 week or so)

**Comment by jmini** [comment]
You can get the newest jar in this folder [openapi-generator-cli (3.0.0-SNAPSHOT version)](https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/3.0.0-SNAPSHOT/):
 
For example today:
```
curl https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/openapi-generator-cli-3.0.0-20180528.100103-93.jar
```

**Comment by jimschubert** [comment]
Note that the script only downloads once. You can remove the `if` condition and add `--quiet` to the Maven command to download every newest snapshot.

**Comment by jmini** [comment]
Now that we have published our release [3.0.0](https://github.com/OpenAPITools/openapi-generator/releases/tag/v3.0.0) we are one step closer.

You can consume the artifacts from the "sonatype stage" repository. Add this section in your pom:

```xml:
<repositories>
  <repository>
    <id>sonatype-stage</id>
    <url>https://oss.sonatype.org/service/local/repositories/releases/content/</url>
  </repository>
</repositories>
```

We will soon be on maven central.

**Comment by jmini** [comment]
The artifacts are on maven central.
